Video script

```diff
@@ -0,0 +1,17 @@
+Our Hackathon project is called Allergen Spotter. 
+Do you or someone you love have food allergies? Do you find yourself checking food labels and ingredients lists all the time? Allergen Spotter can help with that. 
+It's a web-based application for spotting allergies on food labels.  
```

Comment:
Need confirmation if we were able to make this a mobile app or if it's only web based.